### ISSUE TITLE
[REQUEST] Second Merchant

### ISSUE THREAD

**Issue by Wintermote** [Blackstone-SS13/BLACKSTONE#444]
## Is your feature request related to a problem? Please describe. 
There's a lack of conflict for a lot of towner roles - outside the court and the bog guard there is often not a lot to do that doesn't feel very doll-drum.

## Describe the solution you'd like
The introduction of a 2nd merchant across from the first, and and end-round triumph goal for each merchant, to make more money than the other merchant.

## Describe alternatives you've considered
A triumph for the merchant if the tax rate stays under a set amount.

## Additional context
Basically the idea is to give overarching, but role-consistent goals for a job like merchant which don't necessary involve killing and murder but can end up that way. Something that encourages them to be a little more politically active and drive intrigue and player engagement with other players.

Ideally every role would have these kinds of goals that spur people into conflict and competition. At the moment, I feel that only the court and the bog guard have these elements in place.

Also a 2nd merchant would be nice just for sake of eliminating the lineups.

Might be an idea to do the same with the armor/weapon smiths - turn into 2 competing smithies across from each other with their own forges and store brands.

**Comment by OrbitalParadise** [comment]
2nd merchant is too much bloat because then they too would need shophands, maybe there can be a work around with making a market place spot in town that can make competition with the merchant by simply bartering. Another idea is a port master/trader that has a small shop in the market with no shop hands.

**Comment by Wintermote** [comment]
>2nd merchant is too much bloat because then they too would need shophands

Yeah, that's a concern I had too. You don't want too many roles eaten up on this, but at the same time I think shophands have been increased to 3 slots now anyway? 2 merchants + 2 shophands would be the same as 1 merchant + 3 shophands.

**Comment by YosemiteYam** [comment]
You don't need a second merchant, what we need is more unique townie roles that can be craftsmen/entertainers/minor tradesmen. Shit like tailors, leatherworkers, shoe-makers. That way you have organic competition in trade and the merchant isn't the only person selling things, but you don't do that by just adding a 1-1 copy of the merchant when the role is literally supposed to have a monopoly on merchanting as the represantative of the guild. Basically theres 1 "Wall-Mart" versus the local specialized craftsmen like the smiths/tailors etc.
